ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -132,6 +143,17 @@ spec:
               [service]
               service.stalld=start,enable
               service.chronyd=stop,disable
+    # # AmqInstance is required if PTP and BMER operators use AMQP transport
+    # - fileName: AmqInstance.yaml
+    #   policyName: "config-events-policy"
+    # - fileName: HardwareEvent.yaml
+    #   policyName: "config-events-policy"
+    #   spec:
+    #     nodeSelector: {}
+    #     transportHost: "amqp://amq-router.amq-router.svc.cluster.local"
+    #     # transportHost: "http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:9043"
+    #     # storageType: "storage-class-http-events"
+        logLevel: "debug"
```

Comment:
It looks like this line is updating the TunedPerformancePatch. Is that intentional?

Reply:
good catch, commented out since is part of HardwareEvent.yaml declaration.